ProtASR is an evolutionary framework to reconstruct ancestral protein sequences accounting for structural constraints.

It is known that protein evolution is influenced by the protein structure. However, most of current phylogenetic frameworks do not implement structurally constrained substitution models of evolution due to their mathematical complexity.

We have recently developed a set of structurally constrained substitution (SCS) models of protein evolution that consider positive and negative design and that can generate site-specific evolutionary parameters to a likelihood function. We already found that this model can better fit real protein evolution with respect to the traditional empirical substitution models in terms of maximum likelihood (evaluated with the AIC criterion) and amino acid distribution across sites.

ProtASR arises from these aims and consists of a fast and accurate evolutionary framework to infer ancestral protein sequences accounting for structural constrains. ProtASR, through the implemented SCS models, can generate ancestral proteins that are more stable than proteins generated with the classical empirical substitution models.

To download ProtASR we recommend use the Chrome browser. Then go to releases and click on the desired files. The latest release is ProtASR2.2.zip, which outperforms the previous releases and include more SCS models (among other aspects). Do not download Source code files because they only include a readme.


References
Old version of ProtASR: Arenas M, Weber CC, Liberles D & Bastolla, U. 2017. ProtASR: An Evolutionary Framework for Ancestral Protein Reconstruction with Selection on Folding Stability. Systematic Biology, 66(6):1054-1064.

New version of ProtASR (ProtASR2.2): Manuscript in preparation.


Acknowledgments
MA wants to thank the Spanish Government through the grant “Ramón y Cajal” RYC-2015-18241.


Help
Do not hesitate to contact us (miguelmmmab@gmail.com) for any question.
